src/page/ShippingStep.page.ts

```diff
@@ -0,0 +1,19 @@
+import { $, ElementFinder, promise } from 'protractor';
+
+export class ShippingStepPage {
+  private get termsOfServiceCheckbox(): ElementFinder {
+    return $('#cgv');
+  }
+
+  private get checkoutButton(): ElementFinder {
+    return $('#form > p > button > span');
+  }
+
+  public agreeTOS(): promise.Promise<void> {
```

Comment:
group the two actions in the same method, remember you use async/await

Reply:
done